docs/pages/api/getting-started.mdx

```diff
@@ -101,7 +101,7 @@ func main() {
 EOF
```

Comment:
Should EOF be part of this code block?  When EOF is left in, I get this error:
`./main.go:40:1: syntax error: non-declaration statement outside function body`

Also, it's probably my fault, but when I remove the EOF and then try `go run main.go` against my test Teleport Cloud instance, I get this:

```
failed to create client: all connection methods failed
	
	lstat /Users/USERNAME/.tsh/keys/USERNAME.teleport.sh/cas: no such file or directory
exit status 1
```

Reply:
EOF isn't a Go keyword, so it creates a compile error. I've suggested putting this in a comment. Did you log in to your cluster with `tsh login` before running main.go?